src/tools/info/tl_info.c

```diff
@@ -41,14 +41,42 @@
         if (ucs_test_all_flags(_cap_flags, \
                                UCT_IFACE_FLAG_##_name##32 | UCT_IFACE_FLAG_##_name##64)) \
         { \
-            printf("#         %12s: 32, 64 bit%s\n", s, domain); \
+            printf("#         %12s: 32, 64 bit%s (deprecated)\n", s, domain); \
         } else { \
-            printf("#         %12s: %d bit%s\n", s, \
+            printf("#         %12s: %d bit%s (deprecated)\n", s, \
                    ((_cap_flags) & UCT_IFACE_FLAG_##_name##32) ? 32 : 64, domain); \
         } \
         free(s); \
     }
 
+#define _PRINT_ATOMIC_POST(_name, _cap, _op, _suffix) \
```

Comment:
can this be partially extracted to a function? to avoid long macro

Reply:
updated